content/it/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Controller
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+Nella robotica e nell'automazione, un _circuito di controllo_ (_control loop_) è un un'iterazione senza soluzione di continuità che regola lo stato di un sistema.
+
+Ecco un esempio di un circuito di controllo: il termostato di una stanza.
+
+Quando viene impostata la temperatura, si definisce attraverso il termostato lo *stato desiderato*. L'attuale temperatura nella stanza è invece lo *stato corrente*. Il termostato agisce per portare lo stato corrente il più vicino possibile allo stato desiderato accendendo e spegnendo le apparecchiature.
+
+{{< glossary_definition term_id="controller" length="short" >}}
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Il modello del controller
+
+Un _controller_ monitora almeno una tipo di risorsa registrata in Kubernetes. 
+Questi [oggetti](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects) hanno una proprietà chiamata *spec* (specifica) che rappresenta lo stato desiderato. Il o i *controller(s)* per quella risorsa sono responsabili di mantenere lo stato corrente il più simile possibile rispetto allo stato desiderato.
```

Comment:
In Italian the plural form of English words is still singular, so: s/controller(s)/controller/